lib/epub_extact.ex

```diff
@@ -0,0 +1,34 @@
+defmodule EpubExtract do
+  alias Floki
+
+  def unzip_and_parse_epub(epub_path) do
+    temp_dir = create_temp_dir()
+
+    {:ok, _} = unzip_epub(epub_path, temp_dir)
+
+    extracted_content = read_temp_dir(temp_dir)
+    parsed_segments = parse_with_floki(extracted_content)
+    :ok = File.rm_rf!(temp_dir)
```

Comment:
not really required to return :ok , since it doesnt give anything meaningful

Reply:
ok